src/sentry/snuba/metrics/extraction.py

```diff
@@ -683,14 +688,41 @@ def _get_args_support(fields: Sequence[str], used_in_function: str | None = None
     if len(fields) == 0:
         return SupportedBy.both()
 
-    if used_in_function == "apdex":
-        # apdex can have two variations, either apdex() or apdex(value).
-        return SupportedBy(on_demand_metrics=True, standard_metrics=False)
+    if support := _configurable_aggregates_supported_by(fields, used_in_function):
+        return support
 
     arg = fields[0]
     return _get_field_support(arg)
 
 
+def _configurable_aggregates_supported_by(
+    fields: Sequence[str], used_in_function: str | None = ""
+) -> SupportedBy | None:
+    supported_by = None
+    # All usage of apdex is on-demand until we can confirm that apdex(300) can be used on alerts
+    # with standard metrics. For details, see test_get_metric_extraction_config_with_no_spec in
+    # tests/sentry/relay/config/test_metric_extraction.py
+    # Remove this block and update tests once confirmed.
```

Comment:
I've created [this ticket](https://github.com/orgs/getsentry/projects/156/views/2?pane=issue&itemId=52833300) to track this comment.